### DISCUSSION TITLE
EQ4 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#76] [Show and tell]
Section: https://teambasedinquirylearning.github.io/precalculus/2024/EQ4.html
Author: @kathypinzon 
Primary Reviewer: @AbbyANoble

**Comment by AbbyANoble** [comment]
For 1.4.2c you are missing the other solution of x=5.

For 1.4.2d, you could make a separate activity instead to realize there are (potentially) two spots on the number line the correct distance away from 0. Reuse |x-7|=2 maybe to point out how to solve, where earlier, they were plugging in and checking answers. I'm thinking something like:

a) Absolute value represents the distance a value is from 0 on the number line. So, |x-7|=2 means that the expression x-7 is 2 units away from 0. What values on the number line could x-7 equal?

- -7
- -2
- 0
- 2
- 7

b) This gives us two separate equations to solve. What are those two equations?
(expecting x-7=-2 and x-7=2, could be a multiple choice)

c) Solve each equation for x. 


When I teach this I find that the more I draw out a number line and point out that | | means distance to 0, the better they catch on to needing two equations because there are two directions we can go. 

![IMG_4778](https://github.com/TeamBasedInquiryLearning/precalculus/assets/85640990/fe58182a-5e11-4016-8f80-b942b7145af1)

**Comment by AbbyANoble** [comment]
[Activity 1.4.4](https://teambasedinquirylearning.github.io/precalculus/2024/EQ4.html#EQ4-3-5) 
 Question marks aren't needed at the end of each problem. 
I like how you included the wrong answer where they subtract before they change the sign of the other number!

Maybe an observation or an aside after this one to clarify why the last one has no solution.

**Comment by AbbyANoble** [comment]
I'd include another example with a greater or equal to see the "or" situation play out. And then some mixed up ones in a fluency type of activity.

**Reply by kathypinzon** [reply]
yes... this was in my outline, I just didn't get to it yet.  Added in now.
